modules/configuration-api/src/main/java/org/apache/ignite/configuration/NamedListChange.java

```diff
@@ -19,7 +19,11 @@
 
 import java.util.function.Consumer;
 
-/** */
+/**
+ * Change closure parameter for the {@link NamedConfigurationTree} instance. Contains methods to modify named lists.
```

Comment:
"Change closure parameter" - sound very ambiguous, can you rephrase it please?

Reply:
Sure